binnacle/lib/DataModel.dart

```diff
@@ -20,13 +21,17 @@ abstract class Boat {
   double _boomAngle = -1;
   double get boomAngle => _boomAngle;
 
-  /// Speed of the boat in miles per hour.
-  double _speed = -1;
-  double get speed => _speed;
-
-  /// Heading of the boat in degrees. Compass direction (North = 0, East = 90)
-  double _headingAngle = -1;
-  double get headingAngle => _headingAngle;
+  /// Position of the boat which includes, course heading,
+  /// ground speed, and location (lat/long). 
+  Position _position;
```

Comment:
Does the position class include definitions for the measurements its variables are in (e.g. MPH for speed)?

Reply:
If you mean in the comments, then yes. 

> 
> 
> Does the position class include definitions for the measurements its variables are in (e.g. MPH for speed)?